de.fu_berlin.inf.dpp.whiteboard/frontend/src/constants/SVG.js

```diff
@@ -0,0 +1,41 @@
+const SVG = {
+  RECT: "rect",
+  CIRCLE: "circle",
+  LINE: "line",
+  ELLIPSE: "ellipse",
+  TEXT: "text",
+  POLYLINE: "polyline",
+  PATH: "path",
+  PROPERTIES: {
+    X: "x",
+    Y: "y",
+    R: "r",
+    D: "d",
+    X1: "x1",
+    X2: "x2",
+    Y1: "y1",
+    Y2: "y2",
+    RX: "rx",
+    RY: "ry",
+    CX: "cx",
+    CY: "cy",
+    WIDTH: "width",
+    HEIGHT: "height",
+    TEXT: "text",
+    COLOR: "color",
+    FILL: "fill",
+    FONT_SIZE: "fontSize"
+  },
+  isNumerical(name) {
+    return name in {
```

Comment:
Why don't you use a Set ?

Reply:
my idea was that dictionaries are faster for look up, but what I forget is that the dictionary is actually re-created every time the function is called. it was a micro optimisation nothing more.